hazelcast-cluster.yaml

```diff
@@ -7,8 +7,6 @@ data:
     hazelcast:
       network:
         join:
-          multicast:
-            enabled: false
           kubernetes:
             enabled: true
```

Comment:
Could you check if it'd still work without the `hazelcast-cluster.yaml` file?

Reply:
It works, I have removed it from the guides